ExeterTeam25/group-software-project
# Changing project name

Done: 
- Have changed the directory names to be UniExplore, and certain files where this file name needs to be referenced. 
- Made it so UniExplore shows at the top of all the pages. 

To Test: 
- Make sure the app runs as normal. 
- View the app name at the top of all the pages.
